### PR TITLE
nftp, new recipes (dependencies + nftp)

### PR DESCRIPTION
**Pull request by Begasus**
nftp (for version 1.72) added, still have to manually cp the initialization files over to ~/.nftp ... could use some help in there as I'm lost on how to set the correct path ...

### CODE REVIEW

File: dev-libs/libmanycode/libmanycode-1.0.0.recipe [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	haiku
+	lib:libasvtools
+	"
+
+PROVIDES_devel="
+	libmanycode_devel = $portVersion
+	devel:libmanycode = 0.0.0 compat >= 0
+	"
+REQUIRES_devel="
+	libasvtools == $portVersion
```
Comment (korli) [inline dev-libs/libmanycode/libmanycode-1.0.0.recipe]: Looks wrong.

Comment (Begasus) [reply dev-libs/libmanycode/libmanycode-1.0.0.recipe]: yeah, missing "base"?

Comment (korli) [reply dev-libs/libmanycode/libmanycode-1.0.0.recipe]: it should be libmanycode

Comment (Begasus) [reply dev-libs/libmanycode/libmanycode-1.0.0.recipe]: Jikes! :)

**Comment by Begasus** [comment]
still wip, will see to accommodate korli into creating one recipe for this instead of multiple recipes, if nothing else it can be closed, I'll keep it up on my fork for people interested ...

**Comment by Begasus** [comment]
I'm guessing this could be closed, no action for a while, I can leave it up on my fork for people interested in it ...

**Pull request closed without merging**